docs/src/man/comparisons.md

```diff
@@ -1,77 +1,208 @@
+| Aggregate by groups             | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean)`            |
+| Rename column after aggregation | `df.groupby('grp')['x'].mean().reset_index().rename(columns={'x': 'mean_x'})`             | `combine(groupby(df, :grp), :x => mean => :mean_x)` |
+| Aggregate and add columns       | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')`                       | `transform(groupby(df, :grp), :x => mean)`          |
+| Aggregate and select columns    | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')[['grp','x_mean','y']]` | `select(groupby(df, :grp), :x => mean, :y)`         |
+
+### More advanced commands
+
+| Operation                 | pandas                                                                       | DataFrames.jl                                             |
+| :------------------------ | :--------------------------------------------------------------------------- | :-------------------------------------------------------- |
+| Complex Function          | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => v -> mean(cos, skipmissing(v)))`       |
+| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`               |
+|                           | `df[['x','y']].mean()`                                                       | `combine(df, [:x, :y] .=> mean)`                          |
+|                           | `df.filter(regex=("^x")).mean()`                                             | `combine(df, r"^x" .=> mean)`                             |
+|                           | `df[['x', 'y']].agg([max, min])`                                             | `combine(df, ([:x, :y] .=> [maximum minimum])...)`        |
+| Multivariate function     | `df.assign(x_y_cor = np.corrcoef(df.x, df.y)[0,1])`                          | `transform(df, [:x, :y] => cor)`                          |
+| Row-wise                  | `df.assign(x_y_min = df.apply(lambda v: min(v.x, v.y), axis=1))`             | `transform(df, [:x, :y] => ByRow(min))`                   |
+|                           | `df.assign(x_y_argmax = df.apply(lambda v: df.columns[v.argmax()], axis=1))` | `transform(df, AsTable([:x,:y]) => ByRow(argmax))`        |
+| DataFrame as input        | `df.groupby('grp').head(2)`                                                  | `combine(d -> first(d, 2), groupby(df, :grp))`            |
+| DataFrame as output       | `df[['x']].agg(lambda x: [min(x), max(x)])`                                  | `combine(:x => x -> (x = [minimum(x), maximum(x)],), df)` |
+
+Note that pandas preserves the same row order after `groupby` whereas DataFrames.jl
+reorders the result according to the grouped keys.
```

Comment:
Note that we don't lose the row order, but since we present rows by group we have to reorder them:
```suggestion
shows them grouped by the provided keys.
```